src/core/core.datasetController.js

```diff
@@ -268,6 +268,39 @@ helpers.extend(DatasetController.prototype, {
 		}
 	},
 
+	/**
+	 * Returns a set of predefined style properties that should be used to represent the dataset
+	 * or the data if the index is specified
+	 * @params {number} index - data index
+	 * @return {IStyleInterface} style object
+	 */
+	getStyle: function(index) {
+		var me = this;
+		var meta = me.getMeta();
+		var element, style;
+
+		if (meta.dataset && index === undefined) {
+			element = meta.dataset;
+		} else {
+			index = index || 0;
+			element = meta.data[index];
+		}
+		style = me._resolveElementOptions(element || {}, index);
+
+		if (style.fill === false || style.fill === null) {
```

Comment:
Why do we need this case?

Reply:
This is for addressing the [problem 7](https://github.com/chartjs/Chart.js/pull/5621#issue-199847518).